.github/workflows/featurenet-create-from-net.yml

```diff
@@ -0,0 +1,158 @@
+---
+name: FE - Create from net
+
+on:
+  workflow_dispatch:
```

Comment:
I think it is enough to have only `workflow_call` in ths repo - it's not intuitive for a user to think what will happen if I trigger workflow from actions; same comment for other workflows in this PR

Reply:
You're right, I have removed `workflow_dispatch` from all the workflow.  Let's just have them includable.